helix-core/src/main/java/org/apache/helix/controller/rebalancer/waged/AssignmentMetadataStore.java

```diff
@@ -76,6 +77,10 @@ public Map<String, ResourceAssignment> getBaseline() {
     return _globalBaseline;
   }
 
+  public boolean hasPersistedLatestBestPossibleAssignment() {
```

Comment:
can you add detail as to why we need this variable?

Reply:
Added a docstring and more comments in code